addons/account/static/src/scss/section_and_note_backend.scss

```diff
@@ -7,7 +7,7 @@ div.oe_kanban_card.o_is_line_note {
     color: #999;
 }
 table.o_section_and_note_list_view tr.o_data_row.o_is_line_section,
-div.oe_kanban_card.o_is_line_section {
+div.oe_kanban_card.o_is_line_section, tr.o_is_line_section {
```

Comment:
line 6 (4 line above), it should probably do the same with line_note:
`div.oe_kanban_card.o_is_line_note {` should become `div.oe_kanban_card.o_is_line_note, tr.o_is_line_note {` or it won't apply the style in front end for note

Reply:
Good catch, I made another frontend file for this (because this one is backend), and I indeed applied the style for note too. Btw the note style is planned to be changed in another task.